src/extension-runners/firefox-android.js

```diff
@@ -384,6 +387,29 @@ export class FirefoxAndroidExtensionRunner {
     await adbUtils.amForceStopAPK(selectedAdbDevice, selectedFirefoxApk);
   }
 
+  async adbArtifactsDir(removeArtifacts?: boolean) {
+    const {
+      adbUtils,
+      selectedAdbDevice,
+    } = this;
+
+    const foundDirectories = await adbUtils.checkOrCleanArtifacts(
+      selectedAdbDevice, removeArtifacts
+    );
+
+    if (foundDirectories) {
+      if (removeArtifacts) {
+        log.info('Old artifacts directories starting with web-ext-artifacts ' +
+                `removed on ${selectedAdbDevice} device`);
+      } else {
+        log.warn(
+          'Old artifacts directories have been found of the target ' +
+          'android device.Use --adb-clean-artifacts to force their removals.'
```

Comment:
Let's also tweak this message to be more consistent with the other one, e.g. `Old web-ext artifacts have been found on ${selectedAdbDevice}. Use --adb-clean-artifacts to remove them automatically.`

(also, please make sure to put a space between '.' and the message that follows it)